apps/server/src/shared/infra/identity-management/keycloak/service/keycloak-configuration.service.ts

```diff
@@ -1,27 +1,31 @@
+import AuthenticationExecutionInfoRepresentation from '@keycloak/keycloak-admin-client/lib/defs/authenticationExecutionInfoRepresentation';
+import AuthenticationFlowRepresentation from '@keycloak/keycloak-admin-client/lib/defs/authenticationFlowRepresentation';
+import ClientRepresentation from '@keycloak/keycloak-admin-client/lib/defs/clientRepresentation';
+import IdentityProviderMapperRepresentation from '@keycloak/keycloak-admin-client/lib/defs/identityProviderMapperRepresentation';
 import IdentityProviderRepresentation from '@keycloak/keycloak-admin-client/lib/defs/identityProviderRepresentation';
 import { Inject } from '@nestjs/common';
-import { System } from '@shared/domain';
-import { SystemRepo } from '@shared/repo';
-import AuthenticationFlowRepresentation from '@keycloak/keycloak-admin-client/lib/defs/authenticationFlowRepresentation';
-import AuthenticationExecutionInfoRepresentation from '@keycloak/keycloak-admin-client/lib/defs/authenticationExecutionInfoRepresentation';
+import { System, SystemTypeEnum } from '@shared/domain';
 import { DefaultEncryptionService, IEncryptionService } from '@shared/infra/encryption';
+import { SystemService } from '@src/modules/system/service/system.service';
+import { SystemDto } from '@src/modules/system/service/dto/system.dto';
+import { SC_DOMAIN } from '../../../../../../../../config/globals.js';
```

Comment:
Importing the env var like this is bad practice.
Better: `Configuration.get('SC_DOMAIN')`
Even better: Add it in `server.config.ts` and retrieve the value from nest's `ConfigService` as in `account.uc.ts`

Reply:
Done